.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: php
+
+php:
+  - 5.6
+  - 7.1
+
+services:
+  - mysql
+  
+env:
+  - DB=mysql
+  
+before_script:
+  # email mock
+  - sudo apt-get install -y -qq postfix  
```

Comment:
this line and the follwing lines are for preventing errors when emails are sent from the application.
http://michaelthessel.com/mock-mail-setup-for-travis-ci/